### ISSUE TITLE
Aura Mastery Bug

### ISSUE THREAD

**Issue by Dibbarra** [dalaranwow/dalaran-wow#5343]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
Aura Mastery currently only doubles the effect of the aura the paladin casting aura mastery is using.
**Expected behaviour**: Tell us what should happen instead
Every paladin in a raid's aura should receive double effect from aura mastery.
**Steps to reproduce the problem**:

1. have 2 paladins, one on frost aura one on shadow.
2. Click aura mastery on frost aura paladin.
3.  Watch as frost aura pally stays at 130 resistance.

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
**Current behaviour**: For some reason Aura mastery can become permanent I have no idea how to recreate this bug but it's been a common thing on this server for a while, I just did some arena games and ran into a paladin with this bug, completely immune to my Wind Shear. 

Here is a video from last year https://www.youtube.com/watch?v=a8KbZaq_Uac&list= you can clearly see no aura mastery is up yet he cannot be interupted.

**Comment by StheNoble** [comment]
i dont know any informations that could support the claim that aura mastery should increase the effect of any aura other then the own.

could OP pls add any prove for that? spell description in wiki/wowhead and ingame tooltip dont seem to support that imo.

**Comment by Dibbarra** [comment]
http://www.wowhead.com/forums&topic=82312/aura-mastery-effectiveness

Somewhere in this discussion they talk about how it doubles every aura, but not talented effects (ie. improved devo aura won't give 3700 armor)

**Comment by Dibbarra** [comment]
@Rushor @Lothloryen Any news on this issue? I think only thing that needs testing is why cyclone prevents aura mastery from being removed.

**Comment by Aridam** [comment]
A speculation I have for that is because Cyclone makes it so that the Paladin using Aura Mastery becomes Immune to everything (Friendly _and_ Hostile actions taken upon the victim) -- result being is they get stuck in that state of being Immune to Silences and Interrupts as when they still had Aura Mastery up -- which could mean that Banish (albeit you can't Banish Paladins), Divine Intervention, and such of the like that make one Immune to everything by the (De)Buff, may cause the same bug.

On another note, Aura Mastery's _only_ supposed to double the effect/give Silence and Interrupt Immunity _only_ to the _same_ Aura used by the _same_ Paladin who uses Aura Mastery -- it is not supposed to double everything and be personal only on Concentration and anything like that, and that is why you have the above-linked tactics used by many Paladins before big nuke-worthy attacks or anything else to switch an Aura for Aura Mastery.

**Comment by Dibbarra** [comment]
@Lothloryen
